linera-views/README.md

```diff
@@ -11,6 +11,134 @@ atomically persisted in storage).
 
 See the [CONTRIBUTING](../CONTRIBUTING.md) file for how to help out.
 
+## Design
+
+# Possible clients
+
+We designed a `KeyValueStoreClient` trait that implements the basic functionalities
+of a ke-value store whose keys are `Vec<u8>` and whose values are `Vec<u8>`.
+
+A number of key-value stores are supported by the system:
+* `memory` is using the memory (and uses internally a simple B-Tree map).
+* `RocksDB` is a disk based key-value store
+* `DynamoDb` is the AWS based DynamoDb service.
+
+For each of those we provide a type that implements the trait.
+
+There are other possible clients that could be added: Redis, Scylladb, Cassandra, etc.
```

Comment:
Words like "possible", "could", "may", "will", etc convey a sense of uncertainty, but in technical documentation, we're mostly interested in facts.

Here, I would not suggest a particular roadmap to add Redis, etc. We can just remove the sentence or express actual design goals:
"The trait KVSC was designed so that more storage solutions can be easily added in the future."

Reply:
Ok, we have several plans. But yes, let us write that only in the GitHub Issues.